packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -141,6 +141,9 @@ setParameters (const Teuchos::ParameterList& List)
     if (relaxType == "Jacobi") {
       PrecType_ = Ifpack2::Details::JACOBI;
     }
+    else if (relaxType == "MTJacobi") {
+      PrecType_ = Ifpack2::Details::MTJACOBI;
+    }
```

Comment:
@ambrad @mhoemmen 

I am fine about the enum notation. In my personal opinion, the parameterlist is exposed to users and more verbose description would be better for someone who is not familiar to the code. If this things are already exposed to users, then we do not change them. Otherwise it will break their code.